native/packages/react-native-bpk-component-spinner/stories.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  View,
+  Platform,
+  ScrollView,
+  StyleSheet,
+} from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+import BpkThemeProvider from 'react-native-bpk-theming';
+import BpkSpinner from './index';
+import { StoryHeading, StorySubheading } from '../../storybook/TextStyles';
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const styles = StyleSheet.create({
+  dark: {
+    backgroundColor: tokens.colorGray900,
+    padding: tokens.spacingBase,
+  },
+  bottomMargin: {
+    marginBottom: tokens.spacingBase,
+  },
+});
+
+const themeAttributes = {
+  spinnerPrimaryColor: '#2d244c',
```

Comment:
This is really being picky but can we have a different colour here, looks like there is no difference between this one  and the dark variant

Reply:
and maybe a token colour rather than hardcoded 😄

Reply:
Soooo with the colour, this came from James. He's aware that it doesn't look very different but asked me to leave it as is, for now at least.

Reply:
A nice thing might be to have some sort of mock theme file that exposes all the necessary theme props for our storybooks, rather than each story file having to define it itself. I figured that's best left for a future PR though.